python/tests/test_preprocess.py

```diff
@@ -0,0 +1,38 @@
+from nose.tools import assert_raises, assert_equal, \
+    assert_false, assert_true, assert_list_equal, \
+    assert_tuple_equal, assert_dict_equal, raises
+import numpy as np
+from pipeline.utils import galvo_corrections
+from scipy.interpolate import interp1d
+
+def test_motioncorrect():
+    """test motion correction"""
+
+    img = np.random.randn(50, 50)
+    xymotion = [2, 5]
+    newim = galvo_corrections.correct_motion(img, xymotion)
+
+    d = np.abs(np.amax(img[xymotion[1]:, xymotion[0]:] - newim[:-xymotion[1], :-xymotion[0]]))
+
+    assert_true(0 < d < 10e-15)
+
+
+def test_rastercorrect():
+    """test raster correction"""
+
+    # img = np.random.randn(512, 512, 1, 1, 100)
+    # raster_phase = 0.1
+    # fill_fraction = 1
+    from pipeline import preprocess
+    from tiffreader import TIFFReader
+    reader = TIFFReader('/Users/titan/data/cache/11676_4_00006_00009.tif')
```

Comment:
Same as above - it's not a good idea to tie your tests to a specific machine. For the test purpose, you can consider moving the `.tif` image directly into the `tests` directory if it's not too large.